web/src/routes/news/containers/TunewsDetailsPage.js

```diff
@@ -74,6 +69,10 @@ export const TunewsDetailsPage = ({ tunewsElement, language, id, city, cities }:
   }
 
   const { title, content, date, eNewsNo } = tunewsElement
+  // Removes Last Update Timestamp in content
+  const filterPattern = `<p>${eNewsNo}</p>`
+  const filteredContent = content.replace(filterPattern,"")
```

Comment:
Sorry, I guess the issue was ambiguous here. It was meant as no dedicated timestamp should be displayed in the Page component, i.e. no last update should be passed to the page component as it is already included in the TüNews content as you noticed correctly. So I'd probably just render the HTML content 1:1 without modifying it like you did as it is external content and may be subject to changes we don't have any influence on in the future. Sorry for the misunderstanding here!

Or was there any other reason why you decided to do it this way?

Reply:
Okay i thought this eNewsNo tag should be removed from content.
Already removed the filter and updated pr.
I mean showLastUpdate text is already set to false, so it should be fine now?